dotnet/cloudbuild.yaml

```diff
@@ -0,0 +1,14 @@
+# Cloud builder for the dotnet builder.
+steps:
+- name: gcr.io/cloud-builders/docker
```

Comment:
please quote all string fields in yaml. I know it works either way, but I'd like all string fields to be consistent.